File: sub-mysql-web-mapper/src/main/java/org/github/base/service/impl/SysRoleServiceImpl.java
package org.github.base.service.impl;

import org.github.base.entity.SysRoleEntity;
import org.github.base.mapper.ISysRoleMapper;
import org.github.base.service.ISysRoleService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 * 角色 服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-21
 */
@Service
public class SysRoleServiceImpl extends ServiceImpl<ISysRoleMapper, SysRoleEntity> implements ISysRoleService {}

File: sub-mysql-mydb-service/src/main/java/org/github/mysql/mydb/base/service/impl/SysRoleServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.mydb.base.service.impl;

import org.github.mysql.mydb.base.entity.SysRoleEntity;
import org.github.mysql.mydb.base.mapper.ISysRoleMapper;
import org.github.mysql.mydb.base.service.ISysRoleService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * 角色表 服务实现类
 *
 * @author JYD_XL
 */
@Service
open class SysRoleServiceImpl : ServiceImpl<ISysRoleMapper, SysRoleEntity>(), ISysRoleService
